server/lib/routes/get-metrics-errors.js

```diff
@@ -29,6 +30,15 @@ function setExtraSentryData(data) {
   if (sentryData) {
     sentryData.release = CONTENT_SERVER_VERSION;
 
+    if (sentryData.stacktrace && sentryData.stacktrace.frames) {
```

Comment:
I am unable to hit this path in the debugger. This [line](https://github.com/mozilla/fxa-content-server/blob/i3167/server/lib/routes/get-metrics-errors.js#L59) always resolves to false when running test. Might have to mock sentry config?